src/Support/Support.Autorest/test/Update-AzSupportCommunication.Tests.ps1

```diff
@@ -14,9 +14,14 @@ if(($null -eq $TestName) -or ($TestName -contains 'Update-AzSupportCommunication
   . ($mockingPath | Select-Object -First 1).FullName
 }
 
-Describe 'Update-AzSupportCommunication' {
-    It 'UpdateExpanded' -skip {
-        { throw [System.NotImplementedException] } | Should -Not -Throw
+Describe 'Update-AzSupportCommunication' {   
```

Comment:
I thought we could only get and create a new communication-not update it right? We can hide/remove Update-AzSupportCommunication and Update-AzSupportCommunicationsNoSubscription in the README

Reply:
That's true we cannot. The test is mocking object to verify that the cmdlet does respond to the model object correctly. Since I could not test for this scenario to ensure the model layer is working as expected, this check helped to validate that layer. But we don't have test scenario to verify end to end behavior

Reply:
Since this feature doesn't exist the cmdlet shouldn't exist-you can remove it from being autogenerated in the README like so: https://github.com/grhuangmsft/azure-powershell/blob/762b4462de8d5780c78c6004c9b4601dd2042476/src/Support/Support.Autorest/README.md?plain=1#L114C2-L117C17

We shouldn't have this test or cmdlet in the first place

Reply:
makes sense. Removed those scenarios and updated README.md